.github/workflows/get_runner_image.yaml

```diff
@@ -0,0 +1,21 @@
+name: Get runner image
+
+on:
+  workflow_call:
+    outputs:
+      runs-on:
+        description: "Image runner"
+        value: ${{ jobs.get-runner-image.outputs.runs-on }}
+
+jobs:
+  get-runner-image:
+    name: Get runner image
+    runs-on: ubuntu-22.04
+    outputs:
+      runs-on: ${{ env.BUILD_ON }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get build-on and run-on values
```

Comment:
We're just getting `build-on` here, so should we update the description to reflect that?